oio/crawler/integrity.py

```diff
@@ -413,7 +416,7 @@ def main():
     t_help = "Element whose integrity should be checked. " \
         "Can be ACCOUNT, ACCOUNT CONTAINER, ACCOUNT CONTAINER CONTENT " \
         "or ACCOUNT CONTAINER CONTENT CHUNK."
-    parser.add_argument('target', metavar='T', nargs='*',
+    parser.add_argument('target', metavar='T', nargs='+',
```

Comment:
Since `account` is now mandatory, can we make it explicit so it appears in help?